### PR TITLE
Automation of create-from-builder-page pipelines

### PR DESCRIPTION
**Pull request by hemantsaini-7**
**Description:**

Automation on pipeline builder page

**Jira Id:** [ODC-6578](https://issues.redhat.com/browse/ODC-6578)

**Pre-conditions/setup:**

**Checks for approving Epic scenarios Automation PR:**

- [x] Execute the @to-do tagged gherkin scripts manually
- [x] Convert the @to-do gherkin scripts to cypress automation scripts
- [x] Once scripts are automated, replace tag @to-do with @epic-number
- [x] Execute the scripts in Remote cluster

**Refactoring criteria for approving Automation PR:**

- [ ] update the test cases count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)

**Execution Commands:**
Example:
```
    export NO_HEADLESS=true && export CHROME_VERSION=$(/usr/bin/google-chrome-stable --version)
    BRIDGE_KUBEADMIN_PASSWORD=YH3jN-PRFT2-Q429c-5KQDr
    BRIDGE_BASE_ADDRESS=https://console-openshift-console.apps.dev-svc-4.8-042801.devcluster.openshift.com
    export BRIDGE_KUBEADMIN_PASSWORD
    export BRIDGE_BASE_ADDRESS
    oc login -u kubeadmin -p $BRIDGE_KUBEADMIN_PASSWORD
    oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
    oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
    ./test-cypress.sh -p pipelines
```

**Screen shots**:

image

**Browser conformance**:

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by sanketpathak** [comment]
Tests are working in my local
Screenshot from 2022-03-23 19-21-24

File: frontend/packages/pipelines-plugin/integration-tests/support/pages/pipelines/pipelineBuilder-page.ts [hunk 1]
```diff
@@ -86,8 +86,9 @@ export const pipelineBuilderPage = {
         detailsPage.titleShouldContain(pageTitle.Pipelines);
         cy.get('body').then(($body) => {
           if ($body.find('[data-test-id= "dropdown-button"]').length > 0) {
-            cy.byLegacyTestID('dropdown-button').click();
-            cy.get(pipelineBuilderPO.pipeline).click();
+            cy.get(pipelinesPO.createPipeline).click();
+            // cy.byLegacyTestID('dropdown-button').click();
+            // cy.get(pipelineBuilderPO.pipeline).click();
```
Comment (jrichter1) [inline frontend/packages/pipelines-plugin/integration-tests/support/pages/pipelines/pipelineBuilder-page.ts:91]: The `Create Pipeline` button becomes a dropdown once pipeline-as-code repositories are available (some of our tests use that). This code was meant to handle that case.

**Comment by hemantsaini-7** [comment]
Closing in favour of #11258

**Pull request closed without merging**